src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,41 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Deadline extends Task {
+    private static final String FORMAT = "deadline {task name} /by {dd/mm/yyyy HHmm}";
+    private LocalDateTime deadline;
+    private static final DateTimeFormatter IN_FORMAT = DateTimeFormatter.
+                                                        ofPattern("dd/MM/yyyy HHmm");
+    private static final DateTimeFormatter OUT_FORMAT = DateTimeFormatter.
+                                                        ofPattern("dd LLL, h:mma");
+
+    public Deadline(String desc, String deadline) {
+        super(desc);
+        setDeadline(deadline);
```

Comment:
I like how this part is done. You have inspired me to have my code perform in this manner instead of taking in a LocalDateTime.